packages/events/ResponderEventPlugin.js

```diff
@@ -497,6 +497,8 @@ const ResponderEventPlugin = {
 
   eventTypes: eventTypes,
 
+  isLegacy: true,
```

Comment:
What does "legacy" really mean here? That it can't handle passive events? Some other constraints?

Reply:
Yes that’s what it means